.github/workflows/create-prerelease-on-tag.yml

```diff
@@ -0,0 +1,33 @@
+name: Create prerelease on tag
+on:
+  push:
+    tags:
+      - "*.*.*.*"
+  workflow_dispatch:
+jobs:
+  build-release-publish:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Validate manifest.json of the extension
+        uses: cardinalby/schema-validator-action@v1
+        with:
+          file: ext/manifest.json
+          schema: "https://json.schemastore.org/webextension.json"
+
+      - name: Setup node
+        uses: actions/setup-node@v1
```

Comment:
Curious why you don't use `actions/setup-node@v3` here

Reply:
Because I copied it from the existing CI, but I will update both occurrences, thanks :+1: